code/modules/client/preference_setup/loadout/lists/eyegear.dm

```diff
@@ -81,11 +81,6 @@
 	path = /obj/item/clothing/glasses/eyepatch/hud/meson
 	cost = 2
 
-/datum/gear/eyes/material
```

Comment:
you've removed material goggles from tbeing a loadout item at all, is that intended?

Reply:
Yes, the meson eyepatch removal is intentional.